cli/internal/run/dry_run.go

```diff
@@ -80,7 +80,8 @@ func DryRun(
 	// Assign the Task Summaries to the main summary
 	summary.RunSummary.Tasks = taskSummaries
 
-	return summary.CloseDryRun(g.WorkspaceInfos, rs.Opts.runOpts.dryRunJSON)
+	// This is terrible, but pass a 0 exitCode into Close()
+	return summary.Close(0, g.WorkspaceInfos)
```

Comment:
Alternative would be to have `Close` take a `*int` so we could pass `nil` here. After recent exit code shenanigans, I'd prefer if we don't have fake successful exit codes getting created (even if they shouldn't ever end up getting out). I think bare minimum we should choose a more obviously fake exit code like -1.

Reply:
my reasoning is:

- the code isn't actually being used downstream (it's not being used by the method, and if it were, the `executionSummary` will not be part of the dry run output)
- even if it was being used, it really _is_ a successful exit of the parent task, so we can see how it's ok. I could use `the `successCode` 

After thinking about it, I think it's the comment I should change/remove rather than the code